### PR TITLE
Add Photo Supreme v3 single user

### PR DESCRIPTION
**Pull request by liljenstolpe**
Install the single-user version of IDImager's DAM package, Photo Supreme v3

### CODE REVIEW

File: Casks/photo-supreme-single-user.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+  url 'http://www.idimager.com/Trial/PhotoSupreme_V3.pkg'
+  name 'Photo Supreme Single User'
+  homepage 'http://www.idimager.com/WP/?page_id=20'
+  license :commercial
+
+  pkg 'PhotoSupreme_V3.pkg'
+
+  uninstall :pkgutil => 'com.idimager.idimagersu'
+
```
Comment (vitorgalvao) [inline Casks/photo-supreme-single-user.rb]: This empty line must go.

Comment (liljenstolpe) [inline Casks/photo-supreme-single-user.rb]: done

**Comment by liljenstolpe** [comment]
made requested changes

**Comment by vitorgalvao** [comment]
Thank you. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 0c2c8000bb364a8f1b59b3c95adb623585d7b189. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**